common/lib/xmodule/xmodule/js/spec/video/video_accessible_menu_spec.js

```diff
@@ -1,290 +0,0 @@
-(function(undefined) {
```

Comment:
Can you explain where these test points went? I assume they weren't all about the items you removed.